Update lambda.md (#1523)
SumoLogic/sumologic-documentation: docs/integrations/amazon-aws/lambda.md

### Collect Amazon CloudWatch Metrics

> * Sumo Logic supports collecting metrics using two source types

Reviewer: ```suggestion
* Sumo Logic supports collecting metrics using two source types:
```

> * Sumo Logic supports collecting metrics using two source types:

Reviewer: I see we have mentioned the same sentence twice, one is with the link and another one is without the link. Why do we have to write it twice?
Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
       or
    * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)

    * Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
       or

>     * Configure an [AWS Kinesis Firehose for Metrics Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Amazon-Web-Services/AWS_Kinesis_Firehose_for_Metrics_Source) (Recommended)

Reviewer: ```suggestion
    * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)
```

:::note
Namespace for **Amazon Lambda** Service is **AWS/Lambda**.
:::

>     * **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account” field.

Reviewer: ```suggestion
    * **Metadata**. Add an **account** field to the source and assign it a value that is a friendly name/alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. The **account** field allows you to query metrics.
```

Reviewer: Can you please check how the "account" field is written in UI? If it's in uppercase, then we should change here as well.

Author: Its mentioned as "account" itself.

Reviewer: @nmohantyctr Thanks for the confirmation.
